tests/features.py

```diff
@@ -0,0 +1,42 @@
+from flask import current_app
+
+
+def enable_feature(name, value=True):
```

Comment:
Sorry, I meant `enable_feature_flag` as opposed to `enable_flag`. Or if we went with your suggestion then `set_feature_flag` rather than `set_flag`.